src/components/common/Footer.vue

```diff
@@ -3,76 +3,47 @@
         </div>
 
-<!--         <div class="footer-content">
-          <div class="mobile-foot-line"></div>
-          <div class="footer-content-title">{{ $t('footer.Cooperation') }}</div>
-          <div class="footer-content-div"><a class="footer-content-text" href="https://info.ont.io/cooperation/en">{{ $t('footer.CooperateWithOntology') }}</a></div>
-          <div class="footer-content-div"><a class="footer-content-text" href="https://info.ont.io/listtriones/en">{{ $t('footer.Nodes') }}</a></div>
-        </div> -->
-
         <div class="footer-content">
           <div class="mobile-foot-line"></div>
           <div class="footer-content-title">{{ $t('footer.About') }}</div>
-<!--           <div class="footer-content-div"><a class="footer-content-text" href="https://info.ont.io/news/en">{{ $t('footer.Press') }}</a></div>
-          <div class="footer-content-div" @click="toTechNews()" ><a class="footer-content-text" >{{ $t('footer.TechNews') }}</a></div>
-          <div class="footer-content-div"><a class="footer-content-text" href="https://info.ont.io/team/en">{{ $t('footer.Team') }}</a></div> -->
-          <div class="footer-content-div"><a class="footer-content-text" href="https://ont.io/#/aboutus">{{ $t('footer.AboutOntology') }}</a></div>
-          <div class="footer-content-div"><a class="footer-content-text" href="https://ontio.github.io/documentation/wp_download_en.html">{{ $t('footer.WhitePaper') }}</a></div>
-<!--           <div class="footer-content-div"><a class="footer-content-text" href="https://explorer.ont.io">{{ $t('footer.Explorer') }}</a></div> -->
+          <div class="footer-content-div"><a class="footer-content-text" href="https://www.onyxcoin.io/index2.php">{{ $t('footer.AboutOntology') }}</a></div>
+          <div class="footer-content-div"><a class="footer-content-text" href="https://wallet.onyxpay.co/">{{ $t('footer.Wallet') }}</a></div>
+          <div class="footer-content-div"><a class="footer-content-text" href="https://www.onyxcoin.io/files/docs/WP.pdf">{{ $t('footer.WhitePaper') }}</a></div>
         </div>
 
         <div class="footer-content">
           <div class="mobile-foot-line"></div>
           <div class="footer-content-title">{{ $t('footer.dApps') }}</div>
-          <div class="footer-content-div"><a class="footer-content-text" href="https://oodapp.io"></a></div>
-          <div class="footer-content-div"><a class="footer-content-text" href="https://oodapp.io/">{{ $t('footer.dApp') }} <img class="wailian-img" src="../../assets/wailian.svg"/></a></div>
-          <div class="footer-content-div"><a class="footer-content-text" href="https://submit.oodapp.io">{{ $t('footer.SubmitAdApp') }} <img class="wailian-img" src="../../assets/wailian.svg"/></a></div>
+          <div class="footer-content-div"><a class="footer-content-text" href="mailto:contact@ont.io">{{ $t('footer.SubmitAdApp') }}</a></div>
```

Comment:
mailto:contact@ont.io Is it correct?

Reply:
no, it's not. we are still waitining for correct contact email address from customers.

Reply:
So we can change it to a dummy value for now